.github/workflows/rs-all-client/CMakeLists.txt

```diff
@@ -0,0 +1,53 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2023 Intel Corporation. All Rights Reserved.
+cmake_minimum_required( VERSION 3.15 )
```

Comment:
This bump our minimal CMake version.
For windows it's less painfull but I believe we can condition it with the version.
<3.15 dont build it?

Reply:
Oh, it's just the test, forget it :)

Reply:
You're right... wouldn't the static analysis catch this?? Weird.
I'll try it on U18.

Reply:
Oh!
Duh...
This is the the GHA client!
It doesn't run on U18...
`realsense2-all` makefile works fine in U18. But this client is made for U20+. That's fine.